Lighthouse Camera/Storyboards/LaunchScreen.storyboard

```diff
@@ -16,12 +20,21 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
```

Comment:
That's correct, launch screen is what first appears when the user taps the app icon before the app is finished launching, xcode now by default create a launchscreen.storyboard for every new project, my only change here is adding a UIImage and assign lighthouse image resource file to it. It displays lighhouse image instead of a blank screen while app is trying to load.
 We might need to build on our launch screen (wireframe 1 in UX spec) depending on more details from UX. 
Main storyboard is what is called when the app is running and handles all the logic.